run/README.namelist

```diff
@@ -514,6 +514,7 @@ Namelist variables for controlling the adaptive time step option:
                                      = 30, HUJI (Hebrew University of Jerusalem, Israel) spectral bin microphysics,
                                            fast version
                                      = 32, HUJI spectral bin microphysics, full version
+                                     = 10, Morrison (2 moments) with consideration of of CESM-NCSU RCP4.5 climatological aerosol 
```

Comment:
@smileMchen 
Ming,
This is supposed to be mp_physics=40, right? You have "10" as the option. Also, there is a repeated "of"

Reply:
@Dave, yes it should be mp_physics =40. Thanks for pointing out the mistake!